html/js/product-preferences.js

```diff
@@ -468,8 +466,10 @@ function display_user_product_preferences(target_selected, target_selection_form
 		);
 
     $('.edit_button.close_food_preferences:last', target_selection_form).before('<div id="external_panels_prefs"></div>');
-    renderExternalPanelsOptinPreferences($(target_selection_form).find('#external_panels_prefs')[0]);
-
+    const prefEl = $(target_selection_form).find('#external_panels_prefs')[0];
+    if (prefEl && typeof window.renderExternalPanelsOptinPreferences === "function") {
+      window.renderExternalPanelsOptinPreferences(prefEl);
+    }
```

Comment:
This does not work because we introduce renderExternalPanelsOptinPreferences later.

Also speaking with @teolemon he told be it would be better if it was a separate button (and cleaner at implementation level).
So I propose that we add it as a separate button, at the same level of preferences but on the left right corner.


<img width="1292" height="382" alt="image" src="https://github.com/user-attachments/assets/dc016ded-7534-4be4-aef3-61705e8e1103" />